src/main/java/runner/Ui.java

```diff
@@ -67,4 +67,24 @@ public static void addMSG(Task t, int n) {
         System.out.println(t);
         System.out.println("Now you have " + n + " tasks in the list.");
     }
+
+    /**
+     * Show the TaskList of a Duke.
+     */
+    public static void showList(TaskList t) {
```

Comment:
In the showList() method, the code to display the task list is duplicated in the findList() method. This can be refactored to a separate method to improve readability and maintainability.